workflows/sanbi_cwltutorial/README.md

```diff
@@ -0,0 +1,167 @@
+## Reverse / Complement in Common Workflow Language
+
+Here are two tools and two workflows, all written to
+demonstrate features of the [Common Workflow Language](http://www.commonwl.org/).
+
+In `reverse/` and `complement/` there are simply Python scripts that implement
+the reverse and [complement](https://en.wikipedia.org/wiki/Complementary_DNA)
+operations on a file containing a DNA string. There are also corresponding
+Docker containers and Dockerfiles for building them.
+
+### reverse: a Common Workflow Language tool description
+
+Each tool has a Common Workflow Language tool description file. The
+file from `reverse.cwl` is reproduced below:
+
+    cwlVersion: v1.0
+    class: CommandLineTool
+
+    hints:
+      DockerRequirement:
+        dockerPull: pvanheus/reverse:latest
+
+    baseCommand: reverse.py
+
+    inputs:
+      infile:
+        type: File
+        inputBinding:
+          position: 1
+
+    stdout: $(inputs.infile.nameroot)\_reversed$(inputs.infile.nameext)
```

Comment:
It was creating a problem with my Markdown preview - escaping it out seemed to work both for display and in the editor.